docs/configuration/rename_function.md

```diff
@@ -0,0 +1,69 @@
+# How to rename element in architecture object
+
+Architecture objects have callback, node, path, executor, and topic objects.  
+CARET serves APIs to update the names of these objects.
+
+The name of the target element in the architecture object is updated via the Python API.  
```

Comment:
I think same meaning and it is better to delete Line 6.

Reply:
Fixed it in [3aa320b](https://github.com/tier4/CARET_doc/pull/159/commits/3aa320b36cd905375915e4a6cff8ef2ece6ae0ea).